External/FEXCore/Source/Common/StringUtils.h

```diff
@@ -3,7 +3,7 @@
 
 namespace FEXCore::StringUtils {
   // Trim the left side of the string of whitespace and new lines
-  [[maybe_unused]] static fextl::string LeftTrim(fextl::string String, fextl::string TrimTokens = " \t\n\r") {
+  [[maybe_unused]] static fextl::string LeftTrim(fextl::string String, std::string_view TrimTokens = " \t\n\r") {
     size_t pos = fextl::string::npos;
     if ((pos = String.find_first_not_of(TrimTokens)) != fextl::string::npos) {
```

Comment:
Unrelated to optimizing allocations: Shouldn't this be a `while` loop to handle multiple trim tokens?